lib/abr/simple_abr_manager.js

```diff
@@ -345,8 +348,19 @@ shaka.abr.SimpleAbrManager = class {
    */
   getBandwidthEstimate() {
     const defaultBandwidthEstimate = this.getDefaultBandwidth_();
-    const bandwidthEstimate = this.bandwidthEstimator_.getBandwidthEstimate(
+    let bandwidthEstimate = this.bandwidthEstimator_.getBandwidthEstimate(
         defaultBandwidthEstimate);
+
+    // Return the mini(bandwidthEstimate, the network API estimate)
+    // This is relevant ONLY if the data come from the network API (else
+    // It's just a const that effectively disables the ABR) and if the
+    // value is not capped (maxNetApiValue_)
+    if (this.config_.defaultBandwidthEstimate !== defaultBandwidthEstimate &&
```

Comment:
So, essentially, when `navigator.connection.downlink` is giving us a real (e.g. non-capped) estimate, we use the lower of that and of our bandwidth estimator's estimate...

I guess I can understand that logic, but it's a significant change in behavior from how the `useNetworkInformation` configuration value worked previously.
It might be worth gating behind a new configuration value.

Reply:
Which configuration value do you suggest?